bidict/_bidict.py

```diff
@@ -73,10 +79,10 @@ def clear(self):
         self._inv.clear()
 
     def pop(self, key, *args):
-        """Like :py:meth:`dict.pop`, keeping bidirectionality intact."""
-        ln = len(args) + 1
-        if ln > 2:
-            raise TypeError('pop expected at most 2 arguments, got %d' % ln)
+        """Like :py:meth:`dict.pop`, maintaining bidirectionality."""
+        l = len(args) + 1
```

Comment:
Interesting, for me GitHub renders it in Menlo 12px, in which `l` and `1` look quite different.